src/activate/registerCommands.ts

```diff
@@ -310,3 +303,84 @@ export const openClineInNewTab = async ({ context, outputChannel }: Omit<Registe
 
 	return tabProvider
 }
+
+export const openSettingsInNewTab = async ({ context, outputChannel }: Omit<RegisterCommandOptions, "provider">) => {
+	const contextProxy = await ContextProxy.getInstance(context)
+
+	// Get the existing MDM service instance to ensure consistent policy enforcement
+	let mdmService: MdmService | undefined
+	try {
+		mdmService = MdmService.getInstance()
+	} catch (error) {
+		// MDM service not initialized, which is fine - extension can work without it
+		mdmService = undefined
+	}
+
+	const tabProvider = new ClineProvider(context, outputChannel, "editor", contextProxy, mdmService)
+	const lastCol = Math.max(...vscode.window.visibleTextEditors.map((editor) => editor.viewColumn || 0))
+
+	// Check if there are any visible text editors, otherwise open a new group
+	// to the right.
+	const hasVisibleEditors = vscode.window.visibleTextEditors.length > 0
+
+	if (!hasVisibleEditors) {
+		await vscode.commands.executeCommand("workbench.action.newGroupRight")
+	}
+
+	const targetCol = hasVisibleEditors ? Math.max(lastCol + 1, 1) : vscode.ViewColumn.Two
+
+	// Get the localized title for the settings tab
+	const settingsTitle = t("settings:tabTitle") || "Roo Code Settings"
+
+	const newPanel = vscode.window.createWebviewPanel(
+		"rooCodeSettings", // Unique ID for settings panel
```

Comment:
Using a different viewType ("rooCodeSettings") breaks title bar menu conditions that check for the main panel type (ClineProvider.tabPanelId). This hides commands when the settings tab is active. Use the same type to retain consistent UX across tabs.

```suggestion
ClineProvider.tabPanelId,
```